internal/scripting/script.go

```diff
@@ -78,3 +99,47 @@ func (l *Loader) watch(uriOrCode string, onUpdate func(io.Reader) error) error {
 	// Perform a first update
 	return onUpdate(bytes.NewReader(u.Data))
 }
+
+// Load load go plugin URL and starts a watching if needed
+func (l *PluginLoader) LoadGoPlugin(name, uriOrCode string) (*GoFunc, error) {
+	log.Println("LoadGoPlugin: ", name, uriOrCode)
+	// try to download it
+	if err := l.watch(uriOrCode, l.updateGoPlugin); err != nil {
+		return nil, err
+	}
+
+	return newGoFunc(name, l.main), nil
+}
+
+func (l *PluginLoader) updateGoPlugin(r io.Reader) error {
+	l.functionName = "ComputeRow"
```

Comment:
Function name should come from the configuration, we should be able to support  multiple functions in a single library.